### DISCUSSION TITLE
Feature request: interactive shell

### DISCUSSION THREAD

**Discussion by revmischa** [logandk/serverless-wsgi#208] [Ideas]
Would love to be able to get an interactive shell. Right now it just quits immediately.

```
❯ sls wsgi flask --command shell
Python 3.7.5 (default, Dec 27 2019, 22:56:11) 
[GCC 4.8.3 20140911 (Red Hat 4.8.3-9)] on linux
App: MyApp [production]
Instance: /var/task/instance
>>> 
now exiting InteractiveConsole...
```

**Comment by logandk** [comment]
Indeed, this would be a nice feature. It would involve either:

- Serializing the state of the shell after each command (likely to break in a number of ways)
- Using a websocket

I think websockets would be the right approach, but it's not something I'm going to be working on at the moment. I'm happy to accept a PR, though.
